jsonschema-core.md

```diff
@@ -1645,27 +1645,23 @@ and the instance value are affected by this keyword.
 This keyword produces an annotation value which is the largest index to which
 this keyword applied a subschema. The value MAY be a boolean true if a subschema
 was applied to every index of the instance, such as is produced by the `items`
-keyword. This annotation affects the behavior of `items` and `unevaluatedItems`.
+keyword.
+
+The presence of this keyword affects the behaviors of [`items`](#items) and
+[`unevaluatedItems`](#unevaluateditems).
 
 ##### `items` {#items}
 
 The value of `items` MUST be a valid JSON Schema.
 
-This keyword applies its subschema to all instance elements at indexes greater
-than the length of the `prefixItems` array in the same schema object, as
-reported by the annotation result of that `prefixItems` keyword. If no such
-annotation result exists, `items` applies its subschema to all instance array
-elements.[^11]
-
-[^11]: Note that the behavior of `items` without `prefixItems` is identical to
-that of the schema form of `items` in prior drafts. When `prefixItems` is
-present, the behavior of `items` is identical to the former `additionalItems`
-keyword.
+This keyword applies its subschema to all instance elements at indices greater
+than the length of the `prefixItems` array in the same schema object. If
```

Comment:
> json arrays are indexed from 0

[JSON 8259](https://datatracker.ietf.org/doc/html/rfc8259#section-5) doesn't specify and indexing base.  Seems more like a programming language thing to me.

I'll see if I can fix this so that it doesn't mention indices at all.

Reply:
Maybe 

> This keyword ignores instance element in the array, up to and including the number of items found in the `prefixItems` array in the same schema object. It then applies its subschema to all subsequent instance elements.

Reply:
https://www.rfc-editor.org/rfc/rfc6901.html#section-4 specifies that JSON pointer indexes are zero-based.

I don't see how we can avoid indexes because that's a necessary part of the annotation content for these keywords (and `contains` contains a list of indexes, not just the number of evaluated/validated elements), and we also use array indexes in any JSON pointer in a `$ref`, and in keywordLocations and absoluteKeywordLocations in error objects.

Reply:
That's a very good point.  I think it's important to state that we're using zero-based indexing and reference that section.

I'll work that in.  Thanks for the... pointer 😏

Reply:
I've left this in as-is (without references to indexing), but I've also added a section specifically about array indices.